### PR TITLE
Lightning completion refinement

### PR DESCRIPTION
**Pull request by lushang**
1. Update aura:handler attributes and scope for is_lightning
2. Fix bug for Lightning component interface completion

### CODE REVIEW

File: completions.py [hunk 1]
```diff
@@ -722,7 +722,7 @@ def on_query_completions(self, view, prefix, locations):
         #   e.g. force:appHostable, force:lightningQuickAction
         if ch in ['"', ',', ' ']:
             matched_region = view.find('(implements="[\\w\\s:\\,]+")', begin)
-            if matched_region and matched_region.contains(pt): 
+            if matched_region and pt < matched_region.b:
```
Comment (xjsender) [inline completions.py]: It failed to pass validation, @lushang

File: completions.py [hunk 2]
```diff
@@ -459,7 +459,7 @@ def on_query_completions(self, view, prefix, locations):
 
         # Determine if current file is Lightning
         file_name = view.file_name()
-        is_lightning = True if file_name.split(".")[-1] in ["app", "cmp"] else False
+        is_lightning = True if file_name.split(".")[-1] in ["app", "cmp", "evt"] else False
```
Comment (xjsender) [inline completions.py]: @lushang do we need to add any base component in the event?

**Comment by xjsender** [comment]
@lushang , please help to review the latest version, because I add SLDS support to haoide, thanks.

**Comment by xjsender** [comment]
@lushang  I have added you to collaborator.

File: completions.py (hunk 1 above)
Comment (lushang) [reply completions.py]: Let me check it again. any better way to fix this issue?

File: completions.py (hunk 2 above)
Comment (lushang) [reply completions.py]: <aura:attribute> may be used in event. So I add in the list to support this situation

Comment (xjsender) [reply completions.py]: @lushang , you are right, please pull a new request based on latest version.

File: completions.py (hunk 1 above)
Comment (xjsender) [reply completions.py]: Let it go firstly, if you have better solution, please pull.

File: completions.py (hunk 2 above)
Comment (lushang) [reply completions.py]: No problem

**Pull request closed without merging**